scripts/apis_scan.py

```diff
@@ -161,8 +163,19 @@ def enable_passive_scanner():
     zap.pscan.enable_all_scanners()
     zap.pscan.disable_scanners(disabledPassiveScan)
 
+def importurls(filepath):
+    """
+    Imports URLs (one per line) from the file with the given file system path.
+    This component is optional and therefore the API will only work if it is installed
+    """
+    return six.next(six.itervalues(zap._request(zap.base + 'exim/action/importUrls',
+                                                         {'filePath': filepath})))
 
 def get_APIs():
+    if urlScan:
+        url_list_path = "/zap" + urlScanDir + "/" + "urlScan.config"
+        print ("Scanning from URL List")
```

Comment:
Would you mind to use `logging` instead of `print` statements? It would give us an opportunity to better maintain the project

Reply:
Sure 👍

Reply:
Thank you! Yep, I think it requires its own PR